src/governance.rst

```diff
@@ -144,6 +144,25 @@ the time when a vote is called, but may never be less than 24 hrs.
     * Sensitive
     * 75% Majority to pass
 
+:Overall workflow and packaging policies: The proposer can choose to
+    create a poll with an external tool or call
+    for voting on the GH issue in question.
+    The voting must period be open for at least one core
+    member meeting cycle to allow for clarification questions
+    and discussions.
+
+    * Standard
+    * 50% Majority to pass
```

Comment:
What about a threshold in terms of number of users out of the org participating?

Reply:
You can't force people to vote.  If anything, we should have rules about where polls should be posted, and how many reminders should be posted.  Beyond that, if people still don't participate, it should not invalidate a poll.

Reply:
My concern is having enough buy-in for big changes. Otherwise they will be very hard to enact (even if a handful of people agree with them unanimously) or for that matter maintain.

Reply:
If it's not going to attract enough attention to get some minimum number of votes, why does it need a vote at all? That suggests to me at least that the question isn't interesting and those involved can reasonably resolve it through a friendly discussion.

Reply:
Just take a look at the number of things we discuss in many meetings that do not attract attention and yet are blocked. Sorry, but the reality is not the scenario you described above. Also, why not vote after a discussion? Isn't that better than deciding on a closed meeting?

Reply:
Based on my experience in our faculty senate and on several committees, I'd suggest a couple minor clarifications:

+ Let's clarify what "50%" (and 75% below) mean: I'd recommend changing the language to "50% of those voting to pass" and "75% of those voting to pass".
+ Maybe change "50%" to "50% plus one" so that when we have a tie vote we know what to do. If we don't decide this now then Murphy's law dictates we'll hit the situation in the first vote 😀 
+ @jakirkham is essentially suggesting we define what constitutes a quorum. That works very well for in-person meetings (and is arguably essential for good governance in that case); I understand the problem with compelling participation online, though. So how about this: `In the event that fewer than 50% of core members have participated in a vote by 24 hours before the deadline a reminder will be sent that they must vote to register approval or disapproval of the proposal. Failing to vote by the deadline will not affect the outcome.` This would set a goal of 50% participation, but not a block in the event we just don't get the responses. My guess in the event of very low participation is that most simply don't have an opinion (or don't understand the details of the proposal well enough to comment, at least in my case sometimes). I'm guessing those with strong opinions either way will vote.

Reply:
> + Let's clarify what "50%" (and 75% below) mean: I'd recommend changing the language to "50% of those voting to pass" and "75% of those voting to pass".

Good catch.

+ Maybe change "50%" to "50% plus one" so that when we have a tie vote we know what to do. If we don't decide this now then Murphy's law dictates we'll hit the situation in the first vote 😀 

:+1: 

> + @jakirkham is essentially suggesting we define what constitutes a quorum. That works very well for in-person meetings (and is arguably essential for good governance in that case); I understand the problem with compelling participation online, though. So how about this: `In the event that fewer than 50% of core members have participated in a vote by 24 hours before the deadline a reminder will be sent that they must vote to register approval or disapproval of the proposal. Failing to vote by the deadline will not affect the outcome.` This would set a goal of 50% participation, but not a block in the event we just don't get the responses. My guess in the event of very low participation is that most simply don't have an opinion (or don't understand the details of the proposal well enough to comment, at least in my case sometimes). I'm guessing those with strong opinions either way will vote.

The only change I see in your proposal is a burden of sending a reminder. I'm not against it but do we need more chores? We'll have 15 days, a meeting and the GH issues to interact on those during the voting period, do we really need a reminder 24 hours before the deadline?

Reply:
Maybe, "Friendly reminders are encouraged"

Reply:
> Maybe, "Friendly reminders are encouraged"

Sure. We kind of already do that already (at least I do :wink:)but we can write that in here.

Reply:
Yeah, the burden of the reminder is a bit of a pain (and if it gets forgotten, then we'll get to boisterously discuss whether the vote counts if the reminder is forgotten).

What about clarifying (in all of the communications about these votes) that: voting yes moves the proposal forward; voting no is the only way to express opposition to the proposal; not voting  is discouraged, but  non-votes do not count as "no".

We could maybe add an explicit "Abstain" option to voting; I would choose that in some cases if it were available and it would give me or others a way to formally vote even if there are no strong feelings.

The problem with simply not voting is that "I never read or thought about the proposal or totally forgot to express an opinion" is indistinguishable from "I read it and don't have strong feelings either way".

Providing an "Abstain" option allows that, and it would be sensible to count those towards the "quorum".

Reply:
> We could maybe add an explicit "Abstain" option to voting; I would choose that in some cases if it were available and it would give me or others a way to formally vote even if there are no strong feelings.

Indeed that was missing and we must add that. Thanks!